components/table/GenerateColumns.razor.cs

```diff
@@ -56,6 +56,11 @@ protected override void BuildRenderTree(RenderTreeBuilder builder)
                 if (HideColumnsByName.Contains(property.Name)) continue;
                 var columnType = typeof(Column<>).MakeGenericType(property.PropertyType.GetUnderlyingType());
                 var instance = Activator.CreateInstance(columnType) as IFieldColumn;
+                
+                if (instance != null) 
+                {
+                    instance.ColIndex = i / 3;
```

Comment:
Why the ColIndex is i / 3?

Reply:
for every column i is incremented thrice as I see in this loop, and col index should be i / 3 because of this.

Reply:
It's better to define another varibale for col index